src/wp-includes/blocks.php

```diff
@@ -71,6 +71,14 @@ function remove_block_asset_path_prefix( $asset_handle_or_path ) {
  * @return string Generated asset name for the block's field.
  */
 function generate_block_asset_handle( $block_name, $field_name ) {
+	if ( strpos( $block_name, 'core/' ) === 0 ) {
```

Comment:
Should probably use Yoda syntax:

```suggestion
	if ( 0 === strpos( $block_name, 'core/' ) ) {
```